cmd/globals.go

```diff
@@ -190,6 +190,10 @@ var (
         globalTotalGetIOCount uint64
         globalIsStopping bool = false
         gIsRDDQHandleSet bool = false
+		globalCurrIOCount uint64
+		globalCurrBW uint64
+		globalMetricsChan = make(chan uint32)
+		globalCollectMetrics uint32
 
```

Comment:
For some reason, my editor started using tabs instead of spaces and I couldn't tell since they look the same. I'll fix all the indentation in a separate commit so it doesn't show up with the implementation changes